lib/IRGen/IRGenModule.cpp

```diff
@@ -2040,6 +2040,36 @@ bool IRGenModule::finalize() {
   }
   emitLazyPrivateDefinitions();
 
+  if (!getSILModule().getOptions().StopOptimizationAfterSerialization) {
+    uint32_t ABIValue = 0;
+    if (ObjCInterop) {
+      ABIValue |= 1 << 0;
```

Comment:
What does the shift by 0 do here?

Reply:
my bad, it was supposed to be a pattern of (1 << 0), (1 << 1), (1 << 2), (1 << 3), etc. but I messed it up -- should be clear now